run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+npm install
+npm start
```

Comment:
Consider adding error handling to the script. If an error occurs during the execution of 'npm install' or 'npm start', the script should handle it gracefully.